Simplified/NYPLReadiumViewSyncManager.m

```diff
@@ -194,12 +200,13 @@ - (void)presentSyncReadingPositionAlertForResponse:(NSDictionary *)response
                               NSDictionary *const locationDictionary =
                               NYPLJSONObjectFromData([location dataUsingEncoding:NSUTF8StringEncoding]);
 
-                              NSString *contentCFI = locationDictionary[@"contentCFI"];
-                              if (!contentCFI) {
-                                contentCFI = @"";
-                              }
-                              dictionary[@"openPageRequest"] =
-                              @{@"idref": locationDictionary[@"idref"], @"elementCfi": contentCFI};
+                              NSString *contentCFI = locationDictionary[NYPLBookmarkDictionaryRepresentation.cfiKey];
+
+                              dictionary[@"openPageRequest"] = @{
```

Comment:
I didn't bother about some of the keys because this whole file is R1 only and will be removed